README.md

```diff
@@ -24,29 +36,35 @@ You need go >= 1.7 compiler [see](https://golang.org/doc/install)
 ### Build
 In main directory of this repository do:
 
-    make build
+```
+make build
+```
 
 ### Test
 
-    make test
-
-## Usage of akubra:
+```
+make test
+```
 
-    usage: akubra [<flags>]
-
-    Flags:
-          --help       Show context-sensitive help (also try --help-long and --help-man).
-      -c, --conf=CONF  Configuration file e.g.: "conf/dev.json"
+## Usage of Akubra:
 
+```
+usage: akubra [<flags>]
 
+Flags:
+      --help       Show context-sensitive help (also try --help-long and --help-man).
+  -c, --conf=CONF  Configuration file e.g.: "conf/dev.json"
+```
 
 ### Example:
 
-    akubra -c devel.json
+```
+akubra -c devel.json
+```
 
 ## How it works?
 
-Once request comes to our proxy we copy all it's headers and create pipes for
+Once request comes to our proxy we copy all its headers and create pipes for
 body streaming to each endpoint. If any endpoint returns positive response it's
 immediately returned to a client. If all endpoints return error first response
```

Comment:
"return an error, then the first response is passed to the client" (first mention of error, plus here we know that it's the same client from last sentence)